### ISSUE TITLE
AD410 Doorbell Alarm/Siren entity not exposed

### ISSUE THREAD

**Issue by coolhand72** [rroller/dahua#131]
this integration does not seem to expose the alarm entity as suggested in the documentation for the AD410 doorbell, everything else seems to work fine, including the strobe light functionality

**Comment by rroller** [comment]
Ip look into this today and get back with you

**Comment by rroller** [comment]
I'm not able to find the API to control the siren yet. Still digging.

**Comment by rroller** [comment]
Having a hard time finding the APIs to trigger the siren. Anyone else able to find it?

**Comment by GaryOkie** [comment]
Nope.   I actually opened an Amcrest Support ticket asking specifically about the Siren API, and after several back and forths, got this absurd answer...

"_I was informed by the developers that this doorbell currently does not accept programable commands, and integrating that would require developing an entirely new firmware for the device. 

In short, it looks like the siren command does not exist and is unlikely to be implemented I'm afraid._ "

What concerns me is this may be what to expect in a forthcoming firmware update - a complete neutering of the http server and API.    This appears to have already occurred with the equivalent Lorex doorbell and the latest firmware update to the Dahua DB6I.   Reports I've seen say they do NOT accept API commands.

Oh - there is a way to simulate a siren on the AD110/AD410.    Send a compatible G117 audio file that is long and loud of whatever siren sound you want.   The API for this is documented under 4.3.3 Post Audio Stream.    I've tested it sending an appropriately converted sound file with Home Assistant using curl system command and it works just fine.   It could be built in to the integration.

**Comment by zios0007** [comment]
A siren entity exists, as `DahuaSirenBinarySwitch`. It is named "Siren" on a direct camera and "Alarm" on an NVR channel.

Whether you get one is the awkward part, and it is the reason issues like this keep being filed five years later: detection is partly by **matching the model name**, so hardware that has a siren gets no entity if nobody added the model to the list.

That is #676, and @jays3l33t has #717 open to detect it over RPC2 instead, including on cameras where the CGI endpoint is missing entirely.

For an AD410 specifically, `is_amcrest_doorbell()` recognises `AD` models, so it should be detected. If yours has no siren entity on a current version, that is worth knowing and the diagnostics download says why: integration page, three dots, **Download diagnostics**, then look under `capabilities`. It is redacted, and it now reports which tables your device refused as well as what it supports.

@rroller this can be closed in favour of #676 unless @coolhand72 still has the AD410 and still sees nothing.
